app/src/types.ts

```diff
@@ -0,0 +1,81 @@
+import { Dispatch, SetStateAction } from 'react'
+import { UrlObject } from 'url'
+
+/*
+ * Types relating to persistent user data
+ */
+export type Clinic = {
+  id: number
+  agency: string
+  agencyAddress: string
+  agencyTelephone: string
+  clinic: string
+  clinicAddress: string
+  clinicTelephone: string
+  county: string
+  zip: string
+}
+
+export type ChooseClinicData = {
+  clinic: Clinic | undefined
+  zipCode: string
+}
+
+export type ContactData = {
+  firstName: string
+  lastName: string
+  phone: string
+  comments: string
+}
+
+export type EligibilityData = {
+  residential: string
+  categorical: string[]
+  previouslyEnrolled: string
+  adjunctive: string[]
+}
+
+export type IncomeData = {
+  householdSize: string
+}
+
+export type SessionData = {
+  chooseClinic: ChooseClinicData
+  contact: ContactData
+  eligibility: EligibilityData
+  income: IncomeData
```

Comment:
Notes for reviewers: Collects all the user-inputted data submitted in the form wizard page and is stored in session storage for the duration of a user "session". Because this is stored in session storage, rather than local storage, it persists through page refreshes, but is deleted when the browser/tab is closed, supporting a project goal to protect user data as much as possible. 

This data is also explicitly cleared in 2 circumstances:

1. On `/other-benefits`, when the user clicks the "return to start page" button
2. On `/confirmation`, when the user clicks the "start a new application" button

Reply:
just to confirm, the `/other-benefits` is the page that appears in the event that the user is found *not* eligible for WIC based on their inputted data